indico/modules/events/editing/blueprint.py

```diff
@@ -16,4 +16,7 @@
 
 _bp.add_url_rule('/editing/api/file-types', 'api_file_types', backend.RHEditingFileTypes)
 _bp.add_url_rule('/editing/api/tags', 'api_tags', backend.RHEditingTags)
-_bp.add_url_rule('/editing/api/editable/<int:editable_id>', 'api_editable', backend.RHEditable)
+
+_bp.add_url_rule('/api/contributions/<int:contrib_id>/editing/<type>', 'api_editable', backend.RHEditable)
+_bp.add_url_rule('/api/contributions/<int:contrib_id>/editing/<type>', 'api_create_editable',
+                 backend.RHCreateEditable, methods=('PUT',))
```

Comment:
Why `PUT` here? Wouldn't `POST` be more suitable?

Reply:
I think both are valid here; with POST I'd rather add a `/create` at the end since especially once there are the other urls, just POSTing to that url won't be super clear.